guides/v2.2/magento-functional-testing-framework/release-2/introduction.md

```diff
@@ -61,24 +60,22 @@ As a software engineer, perform regression testing before release to ensure that
 
 ## Find your MFTF version
 
-The MFTF is installed as a Composer dependency in `<magento_root>/app/code/<vendor_name>/<module_name>/Test/Mftf/composer.lock`.
-
-Open the file and check the corresponding dependency:
-
-```json
-{
-    ...
-    "packages": [
-        ...
-        {
-            "name": "magento/magento2-functional-testing-framework",
-            "version": "2.3.0",
-            ...
-        }
-        ...
-    ]
-    ...
-}
+### Using the MFTF CLI
```

Comment:
Imperatives can be perceived as a sequence of steps instead of choice. 
I'll rephrase to:

```
## Find your MFTF version

There are two options to find out your MFTF version:
- using the MFTF CLI
- using the Composer CLI

### MFTF CLI

...

### Composer CLI

...
```